integration-test/src/main/java/com/datastrato/gravitino/integration/test/MiniGravitino.java

```diff
@@ -152,13 +157,23 @@ public void stop() throws IOException, InterruptedException {
     LOG.debug("MiniGravitino shutDown...");
 
     executor.shutdown();
-    Thread.sleep(500);
+    try {
+      Thread.sleep(500);
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+      throw e;
+    }
     executor.shutdownNow();
 
     long beginTime = System.currentTimeMillis();
     boolean started = true;
     while (System.currentTimeMillis() - beginTime < 1000 * 60 * 3) {
-      Thread.sleep(500);
+      try {
+        Thread.sleep(500);
```

Comment:
ditto.

Reply:
hi @qqqttt123 , I just review the code again, considering the code logic needs to handle the interrupt exception and re-throw the exception, so I don't think 'sleepUninterruptibly' can meet the requirement. Can you confirm it again? thx.

Reply:
MiniGravitino is used for the integration test. The code is mainly to wait for a proper interval. It's more brief and stable to use `sleepUninterruptibly`.

Reply:
OK，you reminded me. I'll simplify it, thx.